src/database/migrations/0019-create-organizations_proposals.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE organizations_proposals (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  organization_id INTEGER NOT NULL,
+  proposal_id INTEGER NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  FOREIGN KEY (organization_id) REFERENCES organizations(id),
+  FOREIGN KEY (proposal_id) REFERENCES proposals(id),
+  UNIQUE(organization_id, proposal_id)
```

Comment:
Ah okay this seems more evidence that this is 1:1. Maybe I should review the subsequent commits, but I hope we aren't 1:1!